config/recipes/beats/README.asciidoc

```diff
@@ -26,6 +26,10 @@ Deploys Filebeat as a DaemonSet with the autodiscover feature enabled. Logs from
 
 Deploys Filebeat as a DaemonSet with the autodiscover feature disabled. Uses the entire logs directory on the host as the input source. This configuration does not require any RBAC resources as no Kubernetes APIs are used.
 
+==== Metricbeat for Elasticsearch and Kibana Stack Monitoring - `stack_monitoring.yaml`
+
+Deploys Metricbeat configured for Elasticsearch and Kibana link:https://www.elastic.co/guide/en/kibana/current/xpack-monitoring.html[Stack Monitoring]. Deploys one monitored Elasticsearch cluster and one monitoring Elasticsearch cluster. You can access the Stack Monitoring app in the monitoring cluster's Kibana. Combine with one of the Filebeat recipes but update the `elasticsearchRef` to point to the `elasticsearch-monitoring` cluster for the full Stack Monitoring experience.
```

Comment:
Should we just add that Filebeat and be done?

Reply:
I don't feel strongly either way. The duplication kind of bothers me but not too much

Reply:
I played with it a bit and I'm more on adding it side. Some reasons:
- we say it's stack monitoring, but even the Stack Monitoring UI is incomplete without it:
<img width="471" alt="Screenshot 2020-07-20 at 07 33 55" src="https://user-images.githubusercontent.com/50632861/87905246-677ae580-ca60-11ea-8f94-212ed45faa02.png">

- I applied the example and then I thought I'm combining it with a Filebeat examples, by applying one too. Took me a while to figure out ES that's being monitored has different labels than in other examples, so Stack Monitoring yaml has to be applied last

- when doing the above I also hit what seems like a [bug](https://github.com/elastic/cloud-on-k8s/issues/3470)

- we already duplicate parts of the config between three Filebeat examples, I think that's ok

It's not that many lines and brings full Stack Monitoring experience, I'd do it :)